### PR TITLE
Xenomorphs EORD respawn

### PR DESCRIPTION
**Pull request by hyper2snyper**
## About The Pull Request

You can respawn as a primordial xenomorph in EORD respawn!

Support for hiveless xenos.

## Why It's Good For The Game

More EORD fun!

## Changelog

:cl:
add: xenomorphs EORD respawn!
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
image

**Comment by shelteredfreak** [comment]
Primordial Bagilite (Kicking Bird) has risen to lead the hive!

Hey, as long as I live long enough to see this go in, all is well.

File: code/modules/mob/living/carbon/xenomorph/castes/king/king.dm [hunk 1]
```diff
@@ -26,15 +26,15 @@
 /mob/living/carbon/xenomorph/king/generate_name()
 	switch(upgrade)
 		if(XENO_UPGRADE_ZERO)
-			name = "[hive.prefix]King ([nicknumber])"	 //Young
+			name = "[hive ? hive.prefix : "Disowned "]King ([nicknumber])"	 //Young
```
Comment (carlarctg) [inline code/modules/mob/living/carbon/xenomorph/castes/king/king.dm:29]: i think it'd be awesome if you changed 'Disowned' to 'Feral' to symbolize how they're wild and hiveless

optional: darken hiveless xenos to differentiate them as well

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by Luxyboo** [comment]
can you make xeno respawn a choice rather than random chance?

**Comment by carlarctg** [comment]
bugs i noticed: your pheromones don't actually buff you, even though you do emit them, presumably because of hive shenanigans

same with weeds - you don't seem to heal from them? i might be wrong

and of course no xeno abilities work in xvx but that's out of scope

**Comment by Ragecrabbo** [comment]
> bugs i noticed: your pheromones don't actually buff you, even though you do emit them, presumably because of hive shenanigans
This is a thing even with non-snowflake xenos in EORD, im guessing the round ending might be the cause of this(?)
> same with weeds - you don't seem to heal from them? i might be wrong
Weeds take 10 seconds of not taking any damage before they start to heal you if you lack recovery pheros because the evil coder cabal wants to oppress poor innocent xenomains.

**Comment by Liznerd** [comment]
eh i prefer to keep it all humans in EORD, Benos just get spawned in the middle of a ton of humans and they get WIPED

**Comment by Durbles** [comment]
After playing a lot of EORDs with this testmerged it's a really fun change. 10/10.

**Comment by FosterGrime-Heymynameslouie** [comment]
are there select xenos that dont get spawned in?
rarely see defilers, carriers and widows

also pheros dont seem to work, they have the outline whenver you select the phero but dosent actually get filled up, I assume this is because of different teamed hives?

cool change 11/5

**Comment by carlarctg** [comment]
it would definitely be great if more xeno types were spawnable, maybe as player preference

im a bit tired of the endless queen and king spam in eord. where are the widows and spitters and bulls! is plasma the issue? give zenos infinite plasma during eord.

**Comment by Lumipharon** [comment]
bxzzzzt

**Pull request closed without merging**